## COMPLEX NETWORK ANALYSIS FOR BIOLOGISTS
This is a one day training course on Network Creation, Visualisation and Partitioning, based on Cytoscape.

## Course Background
Complex natural systems permeate many aspects of everyday life—including human intelligence, social media, biomedicine, agriculture, economics, even our personal and professional relationships. The past decade has seen intensification of research into structural and dynamical properties of complex networks. This course will introduce the basic principles of network theory, and hands-on DIY Network analysis using <a href=https://cytoscape.org> Cytoscape</a> one of the most widely used global platforms for construction and analysis of biomolecular networks such as gene regulatory interactions, protein complexes, hydrogen-bonding meshwork in active sites and neuronal networks. The aim is to conceptualize your own textual, tabular or genomic datasets as networks, and to understand how simple topological features can help to decipher complex properties of systems and processes.

## Course Content & Pre-Requisites
This is a one day long workshop with lectures, Hands-on modules and exercises for use of <a href=https://cytoscape.org> Cytoscape</a> in biological data analysis. The course is expected to be data intensive and participants are required to pre-install the software on their own devices as a pre-requisite.
This course makes use of the Cytoscape 3.8 architecture. 
Much of the content is based on the <a href= http://www.cgl.ucsf.edu/home/scooter/CSB/Introduction_to_Cytoscape.pdf> Cytoscape Manual </a>, highly recommended.  


## Intallation Instructions
Please go to https://cytoscape.org/download.html for downloading and installing the software. This is important to have on your devices before the training course.

If you have any problems with any of these requirements or are at all uncertain about your ability to complete the technical elements of the course, please contact our IT Manager, Paul Judge on pj237@cam.ac.uk. or contact Instructor G.Yadav on gy@nipgr.ac.in

## Course Pages
<a href=/Documents/Set01.md> Morning </a> : Session 1 : Introduction & Set Up

<a href=/Documents/Set02.md> Morning </a> : Session 2 : Creating and Visualizing Networks

<a href=/Documents/Set03.md> Afternoon </a> : Session 3 : Network Partitioning 

<a href=/Documents/Set04.md> Afternoon </a> : Session 4 : Network Aesthetics & Export

## Further reading 

What is Cytocape? https://cytoscape.org/what_is_cytoscape.html

Cline, M., Smoot, M., Cerami, E. et al. Integration of biological networks and gene expression data using Cytoscape. Nat Protoc 2, 2366–2382 (2007). https://doi.org/10.1038/nprot.2007.324 

Kohl et al. (2011) Cytoscape: software for visualization and analysis of biological networks. Methods Mol Biol 696:291-303.(pmid: 21063955)

Shannon et al. (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res 13:2498-504. (pmid: 14597658)

Travel Guide to Cytoscape Plugins: aro Saito, Michael E Smoot, Keiichiro Ono, JohannesRuscheinski,  Peng-Liang Wang, Samad Lotia, Alexander R Pico, Gary D Bader, Trey  Ideker (2012)Nature Methods 9 (11) p. 1069-1076

Mlecnik B, Galon J, Bindea G. Comprehensive functional analysis of large lists of genes and proteins. J Proteomics. 2018 Jan 16;171:2-10. doi: 10.1016/j.jprot.2017.03.016. Epub 2017 Mar 22. PMID: 28343001.

<a href=http://manual.cytoscape.org/en/stable/> Cytoscape 3.8.2 Manual</a>

<a href=https://cytoscape.org/documentation_users.html>Cytoscape Support</a>

<a href=https://cytoscape.org/bug-report.html>Cytoscape Bug Reporting</a>

## Related Courses 

<a href=https://training.csx.cam.ac.uk/bioinformatics/event/3688978>EMBL-EBI: Network Analysis with Cytoscape (Online)</a>

<a href=https://training.csx.cam.ac.uk/bioinformatics/event/3516387>Inferring Co-Expressing Genes and Regulatory Networks from RNA-Seq Data</a>

<a href=https://training.csx.cam.ac.uk/bioinformatics/event/3516410>Identification of Eigen-genes, consensus modules and Network Motifs in co-expression (or other biological) networks.</a>


## Course Schedule and Social Media Tags
* Please find the Workshop program <a href=/Documents/schedule.md>here</a>.

* Please tag @gilienv (Instructor) and use Hashtags <code>#PFH</code> when you tweet or post about the event.

## Instructor

<a href= http://www.nipgr.ac.in/research/dr_gyadav.php> Dr Gitanjali Yadav | University of Cambridge / NIPGR</a>


## Helpers & Volunteers

Ms. Citu, NIPGR 

Dr. Indu Santhangopalan, University of Cambridge

The Volunteers will be part of the workshop and assist with any issues from set up and installation to data checks and course exercises. Please feel free to request them for any support you may wish to have.

# Timetable
09:30 - 11:00	Introduction & Set Up

              Background & Concepts
	      
              Basic Topologies & Case Study
	      
              Set up & Installation

11:00 - 11:15	COFFEE BREAK (15 minutes)

11:15 - 13:00	Creating and Visualizing Networks

              Scale Free Networks
	      
              Data Import, Analysis & Layouts
	      
              Challenge 

13:00 - 14:00	LUNCH BREAK (1 hour)

14:00 - 15:15	Network Partitioning 

		Clustering Strategies 
		
	        DIY Network Communities

15:15 - 15:30	COFFEE BREAK (15 minutes)

15:30 - 17:00	Network Aesthetics

             Tips and Tricks
             	
		Annotate, Download and Export Networks
              
	      	Exercise

17:00 - 17:30	Saving Workspace, Q & A
